modules/web-demo/src/components/WasmMiniscript/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+
+import * as AbstractUtxo from '@bitgo/abstract-utxo';
+
+const defaultDescriptor =
+  'wpkh(xpub661MyMwAqRbcFQpwd6c6aaioiXWuygdeknqE8v6PSNusNjwZypj1uXTVNysfsFPEDL6X3yS1kL6JeWyy9bAiR97Gz8KD9Z1W54uBw9U9j2t/*)';
+
+export default function () {
+  const [descriptorString] = useState(defaultDescriptor);
```

Comment:
NIT: you do not need `useState` for this

Reply:
I wanted to make this more interactive but didn't get around to it